pubspec.yaml

```diff
@@ -59,10 +59,10 @@ dependency_overrides:
   # Overriding due to crash on iOS https://github.com/agnostech/flutter_amplify/pull/12
   flutter_aws_amplify_cognito:
     # path: ../flutter_amplify/flutter_aws_amplify_cognito
-    git:
-      url: https://github.com/orestesgaolin/flutter_amplify.git
-      path: flutter_aws_amplify_cognito
-      ref: 15dfc56
+   git:
+     url: https://github.com/Nash0x7E2/flutter_amplify.git
+     path: flutter_aws_amplify_cognito
+     ref: 5f85b135d06434075d271552d1232c15e3bcd581
```

Comment:
Rebase this onto the branch 'update-dependencies'. It fixes some of the issues with Android crashses.